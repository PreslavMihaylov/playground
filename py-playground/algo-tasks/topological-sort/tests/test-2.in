7
A->B, D
D->C
B->C, E, F
C->
G->D, E
F->
E->
