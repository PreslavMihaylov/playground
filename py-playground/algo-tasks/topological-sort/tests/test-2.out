Topological sort: A, B, F, G, D, C, E
